doc/api/fs.markdown

```diff
@@ -989,6 +989,32 @@ The synchronous version of [`fs.writeFile()`][]. Returns `undefined`.
 
 Synchronous versions of [`fs.write()`][]. Returns the number of bytes written.
 
+## fs.mkdtemp(template, callback)
```

Comment:
This document is sorted in ascending order. Can you please put this in appropriate location?